lib/services/elasticsearch.js

```diff
@@ -336,7 +335,16 @@ class ElasticSearch extends Service {
               }
 
               // The document exits and is active, we reject to prevent the creation
+<<<<<<< HEAD
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Parse error at line 338 column 2:

328:                     return this.refreshIndexIfNeeded(
329:                       esRequest,
330:                       _.extend(res, {_source: request.input.body}));
331:                   })
332:                   .catch(error => Bluebird.reject(
333:                     this.esWrapper.formatESError(error))
334:                   );
335:               }
336: 
337:               // The document exits and is active, we reject to prevent the creation
338: <<<<<<< HEAD
      ^
339:               return this.reject('document_already_exists', esRequest.id);
340: =======
341: 
342: <<<<<<< HEAD
343:               return Bluebird.reject(this.get('document_already_exists'));
344: >>>>>>> f43d54f6... rename getError
345: =======
346:               return Bluebird.reject(errorsManager.get('document_already_exists'));
347: >>>>>>> e629426b... wrap getError in external services
348:             }) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AParsingError)